dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -185,16 +185,18 @@ def pd_progress
   end
 
   def admin_progress
+    require 'cdo/properties'
     SeamlessDatabasePool.use_persistent_read_connection do
-      @user_count = User.count
-      @all_script_levels = Script.twenty_hour_script.script_levels.includes({ level: :game })
+      stats = Properties.get(:admin_progress)
 
-      @levels_attempted = User.joins(:user_levels).group(:level_id).where('best_result > 0').count
+      @user_count = stats['user_count']
+      @levels_attempted = stats['levels_attempted']
       @levels_attempted.default = 0
-      @levels_passed = User.joins(:user_levels).group(:level_id).where('best_result >= 20').count
+      @levels_passed = stats['levels_passed']
       @levels_passed.default = 0
 
-      @stage_map = @all_script_levels.group_by { |sl| sl.level.game }
+      @all_script_levels = Script.twenty_hour_script.script_levels.includes({level: :game})
```

Comment:
It would be possible to cache that call as well. That said, since Properties.get(:admin_progress) is itself a database call, doing so wouldn't remove the database dependency. Given that the non-cached queries are fast, I vote for leaving them as-is.